### PR TITLE
fix: process picker broken on some Linux distros

### PR DESCRIPTION
**Pull request by thenbe**
Fixes https://github.com/microsoft/vscode-js-debug/issues/1752
Also see https://github.com/microsoft/vscode-js-debug/issues/1747

### CODE REVIEW

**Comment by thenbe** [comment]
@microsoft-github-policy-service agree

File: src/ui/processTree/darwinProcessTree.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class DarwinProcessTree extends BaseProcessTree {
    * @inheritdoc
    */
   protected createProcess() {
-    return this.spawn('/bin/ps', [
+    return this.spawn('ps', [
```
Comment (thenbe) [inline src/ui/processTree/darwinProcessTree.ts:46]: I'm aware this is the darwin file, but I figured we might as well be consistent.

**Review by connor4312 — APPROVED** [review]

**Comment by thenbe** [comment]
Yup I see them. I'll have to take a closer look at them on a mac (on my nix machine, those tests and a bunch more are failing even before this commit).

**Comment by connor4312** [comment]
Thanks!

**Review by Tyriar — APPROVED** [review]

**Review by lramos15 — APPROVED** [review]

**Pull request merged**